README.md

```diff
@@ -199,6 +199,27 @@ You can run spotbugs using:
 The spotbugs warnings will be found in `reports/spotbugs/main.html` and `reports/spotbugs/test.html` files in the subproject build
 directories.  Use -PxmlSpotBugsReport=true to generate an XML report instead of an HTML one.
 
+### JMH microbenchmarks ###
```

Comment:
While we're at it we can also mention `-prof gc` and how you should look for the norm allocation rate (since we care about garbage generated per operation rather than per second).

Reply:
I'll provide some text for this soon.

Reply:
Sure. Do you think I should have only the link to jmh-benchmarks/README.md and have all the information on to use it in that page?

Reply:
Let's drop this and include that link before merge.

Reply:
Yeah, I fixed this in my local copy, but I wanted to test it before pushing the changes.

Reply:
Updated.